docs/source/index.rst

```diff
@@ -2,11 +2,27 @@
 .. image:: svg/blaze_med.png
    :align: center
 
-**Blaze** translates a subset of modified NumPy and Pandas-like syntax to
+Blaze
+-----
+
+**Blaze** is a tool that allows Python users a familiar interface to query data living in other data storage systems. It is the next generation of NumPy, Python’s extremely popular array library. Blaze aims to extend the structural properties of NumPy arrays to a wider variety of table and array-like structures that support commonly requested features such as missing values, type heterogeneity, and labeled arrays. Blaze connects a NumPy/Pandas user interface to external databases.  Users can write intuitive high-level queries in Python and have those queries translated to their database of choice.  This frees the user from the intricacies of their database of choice and lets them move freely between technologies.
```

Comment:
Agreed.  That was language to describe our intentions 3 years ago.  At this point, Blaze can be better and more clearly defined.     People should see datashape, array-oriented and table-oriented expressions  and multiple-backends when they see Blaze.